app/src/main/java/org/dosomething/letsdothis/ui/fragments/CategoryFragment.java

```diff
@@ -58,8 +63,7 @@ public void onStart()
     {
         super.onStart();
 
-        EventBusExt.getDefault().register(this);
-        TaskQueue.loadQueueDefault(getActivity()).execute(new ReportBackListTask(position));
+        EventBusExt.getDefault().registerSticky(this);
```

Comment:
I am not sure where I should be registering because it's in a fragment in a viewpager. Sticky so I will never not catch my event.